src/lib/Microsoft.Health.Fhir.R4.Ingest/Service/R4FhirImportService.cs

```diff
@@ -63,17 +64,29 @@ public virtual async Task<string> SaveObservationAsync(ILookupTemplate<IFhirTemp
                 existingObservation = await GetObservationFromServerAsync(identifier).ConfigureAwait(false);
             }
 
-            Model.Observation result;
+            Model.Observation result = null;
             if (existingObservation == null)
             {
                 var newObservation = GenerateObservation(config, observationGroup, identifier, ids);
-                result = await _client.CreateAsync(newObservation).ConfigureAwait(false);
+
+                try
+                {
+                    result = await _client.CreateAsync(newObservation).ConfigureAwait(false);
+                }
+                catch (FhirOperationException ex)
+                {
+                    FhirServiceExceptionProcessor.ProcessException(ex, _logger);
+                }
+
                 _logger.LogMetric(IomtMetrics.FhirResourceSaved(ResourceType.Observation, ResourceOperation.Created), 1);
             }
             else
             {
                 var policyResult = await Policy<Model.Observation>
-                     .Handle<FhirOperationException>(ex => ex.Status == System.Net.HttpStatusCode.Conflict || ex.Status == System.Net.HttpStatusCode.PreconditionFailed)
+                     .Handle<FhirOperationException>(ex =>
+                         ex.Status == System.Net.HttpStatusCode.Forbidden

```

Comment:
I'm not sure I understand why the Forbidden case was added here, I don't expect us to retry for this error?

Reply:
I was trying to catch the FHIR service authorization failure here, but I was wrong.

In commit 4810905, I've reverted this change and updated where to catch the FHIR service authorization failure.